src/pint/observatory/special_locations.py

```diff
@@ -315,8 +330,16 @@ def posvel(self, t, ephem, group=None):
         return geo_posvel + stl_posvel
 
 
-# Need to initialize one of each so that it gets added to the list
-BarycenterObs("barycenter", aliases=["@", "ssb", "bary", "bat"])
-GeocenterObs("geocenter", aliases=["0", "o", "coe", "geo"])
-T2SpacecraftObs("stl_geo", aliases=["STL_GEO"])
-# TODO -- BIPM issue
+def load_special_locations():
+    """Load :class:`~pint.observatory.special_locations.BarycenterObs`, :class:`~pint.observatory.special_locations.GeocenterObs`,
```

Comment:
The first line of docstrings gets used in various summaries, so it's better to push longer stuff down and put a short punchy single-line summary as the first docstring line.

Reply:
OK, updated a bit